docs/user_guide.md

```diff
@@ -25,24 +25,10 @@ It's the repository used for teaching purposes. These are mainly basic agents an
 
 You can find more details about installation [here](installation)!
 
-## Set up an experiment
 - [Environment](environment_page)
 - [Agent](agent_page)
 - [ExperimentManager](experimentManager_page)
 - [Logging](logging_page).
-- [Results analysis & visualization (In construction)]()
-## Experimenting with Deep agents
-- [Torch Agents (In construction)]()
-- [Policy and Value Networks (In construction)]()
-- [Experimenting with Bandits (In construction)]()
-## Reproducibility
+- [Plotting and Statistics](plot_stats_page)
 - [Seeding](seeding_page)
 - [Save & Load Experiment](save_load_page)
-- [Save & Load Agents (In construction)]()
-- [Save & Load Data (In construction)]()
-## Advanced Usage
-- [Custom Agents (In construction)]()
-- [Custom Environments (In construction)]()
-- [Using extrenal libraries](external) (like [Stable Baselines](stable_baselines) and [Gymnasium](Gymnasium_ancor))
-- [Transfer Learning (In construction)]()
-- [AdaStop(In construction)]()
```

Comment:
Could you re-add the links so that we have an idea of where we are going.

Reply:
re added links.